src/StorageDumper/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "PluginConfiguration": {
+    "BlockCacheSize": 1000,
+    "HeightToBegin": 0,
+    "Exclude": [ -4 ]
```

Comment:
what's "Exclude"?

where's this documented?

Reply:
Exclude Contract's ID, same use as the old StatesDumper plugin.
Ignore LedgerContract(-4) storage changes by default.